threadit-web/src/pages/PostThread/PostThread.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import Select from 'react-select'
+import { Button, Card, CardBody, Flex, Input, Stack, Textarea} from "@chakra-ui/react";
+import { PageLayout } from "../../containers/PageLayout/PageLayout";
+import { navStore } from "../../stores/NavStore";
+import { threadStore } from "../../stores/ThreadStore";
+
+export default function PostThread() {
+    const [lockInputs, setLockInputs] = React.useState(false);
+
+    const [title, setTitle] = React.useState('');
+    const [content, setContent] = React.useState('');
+    // const [topic, setTopic] = React.useState('topic-placeholder');
+    // const [spoolId, setSpoolId] = React.useState('spoolid-placeholder');
+
+    // const [threadError, setThreadError] = React.useState('');
+
```

Comment:
This dropdown will most likely be populated with spools that the user is subscribed to

Reply:
I think the dropdown should be removed, and it should just post to the subreddit the user clicks the "Post" button from (once I do spool UI). Or it can be a textbox that searches spool names as you type. But we can change this later when we have more spool related stuff in place.